docs/checkers/index.md

```diff
@@ -2131,6 +2131,17 @@ if __name__ == "__main__":
     name = input()
 ```
 
+By default, [`input`], [`open`] and [`print`] are not allowed. However, you can choose which I/O functions specifically to disallow using the forbidden-io-functions
```

Comment:
Write the option name forbidden-io-functions inside backticks (`) so that it appears in code font.